frontend/src/components/graphs/graph/EchartsDetailGraph.vue

```diff
@@ -997,10 +1000,10 @@ export default class EchartsDetailGraph extends Vue {
 
 <style>
 /*noinspection CssUnusedSymbol*/
-.echarts {
-  width: 100%;
-  height: 100%;
-}
+/*.echarts {*/
+/*  width: 100%;*/
+/*  height: 100%;*/
+/*}*/
```

Comment:
Was this change committed accidentally or does it have to do with the echarts upgrade?

Reply:
A mixture apparently. I wanted to delete it as the new version of vue-echarts finally resized their chart container from 640px to 100% so the code is no longer needed. Apparently I was sidetracked and left the commented out code instead of deleting it completely.

I don't think carrying the CSS rules around as they now don't really do anything anymore is worthwhile. The vue-echarts package listed it as a breaking change, so getting surprised by them reverting it is unlikely.